docs/design/README.md

```diff
@@ -8,6 +8,10 @@ Generally, design docs are on Google docs:
 * [Sync Resource](https://docs.google.com/document/d/1ZbXaEh7v_HcgrRu7N-kEucU2pXyq1Rt7cSmmFOBN2q8/edit)
 * [External Data Response Cache](https://docs.google.com/document/d/1nl6iWMmh9CRT0A6cr-Nkfofv9wp210Bkf6ilLq9LLBo/edit)
 * [Gator SyncSet Support](https://docs.google.com/document/d/1mBvy6Y7TDyZswdYlFBHB3Mi2irLVqga_7ZoZvfPIU0A/edit)
+* [Gatekeeper Enforcement Actions](https://docs.google.com/document/d/1qEzDVcmwPysb3ejvTYKPkHJm3Db3Yr7zShwmgFOsR_k/edit#heading=h.6ewsycnqnfl)
+* [Configuring VAP with Scoped EA](https://docs.google.com/document/d/1u55cMZ4smRAuEZf4BYsSyZOCoOy33Lsgp3xuwDm0l6I/edit#heading=h.9oaxevykg8w6)
```

Comment:
This should be removed in favor of the `Gatekeeper-to-VAP User Flow` doc per this comment https://github.com/open-policy-agent/gatekeeper/pull/3367#discussion_r1581284723